performant-software/DM
# Make frames reorderable in viewer grid through dragging

### What does this PR do?
Adds google closure library DragListGroup functionality to the viewer grid so that viewer containers can be reordered on the fly.

### What issues does it address?
Closes #23 

### How to test
On the [staging site](http://67.207.158.85/), add a varying number of frames to the workspace and try reordering them by clicking and dragging on any space in their title areas that isn't occupied by text or an icon. Switch layouts and add new frames following a rearrangement to confirm content appears correctly.
